codeview/src/types/primitive.rs

```diff
@@ -77,9 +94,125 @@ primitives! {
     (0x0075, T_UINT4, "unsigned __int32"),
     (0x0076, T_INT8, "__int64"),
     (0x0077, T_UINT8, "unsigned __int64"),
+    (0x0078, T_INT16, "128 bit signed int"),
+    (0x0079, T_UINT16, "128 bit unsigned int"),
```

Comment:
Examples of inconsistency with the manual type mappings in `main`.

Looking at the nearby types, the most-consistent thing would be to manually change entries like these descriptions to something like `__int128` and `unsigned __int128`.